test/boost/sstable_compaction_test.cc

```diff
@@ -4946,6 +4946,7 @@ SEASTAR_TEST_CASE(test_large_partition_splitting_on_compaction) {
         position_in_partition::tri_compare pos_tri_cmp(*s);
 
         for (auto& sst : ret.new_sstables) {
+            sst = env.reusable_sst(s, tmp.path().string(), sst->generation().value()).get0();
```

Comment:
Why is this necessary?

Reply:
Quoting from the commit message:
```
Unit tests relying on position metadata were adjusted to enable the loading, such that they can still pass.
```

`reusable_sst` below is changes to pass `{ .load_first_and_last_position_metadata = true }` to `sst->load()`

Reply:
Ah, I missed that.